src/clr-angular/data/datagrid/datagrid.html

```diff
@@ -26,17 +26,17 @@
                 </div>
                 <!-- header for datagrid where you can select one row only -->
                 <div role="columnheader" class="datagrid-column datagrid-select datagrid-fixed-column"
-                     *ngIf="selection.selectionType === SELECTION_TYPE.Single">
+                     *ngIf="selection.selectionType === SELECTION_TYPE.Single" [attr.aria-label]="commonStrings.select">
```

Comment:
My concern here is that the text is not descriptive enough, just saying 'Expand' is confusing especially when there is no actual action that can be taken. I think we need new strings for these cases to explain better what is happening.

Reply:
I also have the same concern and wanted to add some new `commonString` but was not sure what to name it. Maybe "Expand row" or "Show additional content"

PS.
We are talking about `Expand` not `Select` right?!

Reply:
Both cases really. We probably also should allow apps to provide a custom string per row. I'm doing this in the detail pane like so.

https://github.com/vmware/clarity/pull/3338/files#diff-8b30d14d89ae1c6bc80dd53548d75f7dR157

Reply:
This is still a request to support.